umbraco-heartcore/tutorials/querying-with-graphql.md

```diff
@@ -184,7 +184,7 @@ Result:
 
 ### Making the Query Generic
 
-Up until now, we have been working on a single document type which in most cases is fine, but since we want to dynamically fetch content based on the url we can use the `content` field.
+Up until now, we have been working on a single Document Type which in most cases is fine, but since we want to dynamically fetch content based on the url we can use the `content` field.
```

Comment:
to keep the sentence to 25 words and use URL instead of url [(another vale rule)](https://docs.umbraco.com/welcome/contribute/style-guide#names)
```suggestion
Up until now, we have been working on a single Document Type. In most cases this is fine, but since we want to dynamically fetch content based on the URL we can use the `content` field.
```